pom.xml

```diff
@@ -35,6 +35,21 @@
     <test-containers.version>1.21.0</test-containers.version>
 
     <!--sonar configuration-->
+    <sonar-maven-plugin.version>4.0.0.4121</sonar-maven-plugin.version>
+    <sonar.java.binaries>${project.basedir}/target</sonar.java.binaries>
+    <sonar.token>${CODE_ANALYSIS_TOKEN}</sonar.token>
+    <sonar.login></sonar.login>
+    <sonar.password></sonar.password>
+    <sonar.projectKey>uk.gov.companieshouse:company-profile-delta-consumer</sonar.projectKey>
+    <sonar.projectName>company-profile-delta-consumer</sonar.projectName>
+    <sonar.coverage.jacoco.xmlReportPaths>${project.basedir}/target/site/jacoco/jacoco.xml,
+      ${project.basedir}/target/site/jacoco-it/jacoco.xml
+    </sonar.coverage.jacoco.xmlReportPaths>
+    <sonar.jacoco.reports>${project.basedir}/target/site</sonar.jacoco.reports>
```

Comment:
Not sure this section is needed for being picked up by sonar - https://github.com/companieshouse/identity-verification-api/blob/93af013a24d34e52adb4ec91021f2ba5ee4f62c7/pom.xml#L40 doesn't have this but shows in Sonar